spec/locales/en/translation.json

```diff
@@ -50,6 +50,7 @@
     "load": "Load",
     "logo": "Logo",
     "metadataTooltip": "View information/metadata about this object",
+    "mirrorTooltip": "Mirror image",
```

Comment:
Looks like @aeschylus added them last November with https://github.com/ProjectMirador/mirador/commit/dd412143df6c530b4f575c160d05b8255b7f1caa. I think we can remove them in a separate PR.